gslb/ingestion/bootup_handler.go

```diff
@@ -184,8 +184,8 @@ func HandleBootup(cfg *restclient.Config) (bool, error) {
 	}
 
 	if len(amkoClusterList.Items) == 0 {
-		gslbutils.Logf("No AMKOCluster object found, AMKO would start as leader")
-		return true, nil
+		gslbutils.Logf("No AMKOCluster object found, AMKO would stop")
```

Comment:
Can we re-phrase it as `No AMKOCluster object found, rebooting AMKO`?

Reply:
Do we have any bootuptest for it?

Reply:
Updated the log and added test